packages/happy-dom/src/nodes/element/Element.ts

```diff
@@ -793,6 +793,9 @@ export default class Element
 	 * @param name Name.
 	 */
 	public removeAttribute(name: string): void {
+		if (typeof name !== 'string') {
```

Comment:
The name is actually not removed, it is stringified.

You can test it with this:
```javascript
var div = document.createElement('div');

div.setAttribute(null, 'test');

// Outputs '<div null="test"></div>'
console.log(div.outerHTML);

div.removeAttribute(null);

// Outputs '<div></div>'
console.log(div.outerHTML);
```

I think it is best if we add the stringify using `String()` to `NamedNodeMap` to methods such as `getNamedItem()`.